cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt

```diff
@@ -102,27 +105,60 @@ private class NodeRunner(baseDebugPort: Int, baseMonitoringPort: Int) {
     }
 
     private fun startWindowed(command: List<String>, workingDir: File, nodeName: String): Process {
+
+        val createTempScriptFile = {
+            val tmpFile: File = File.createTempFile("corda_node_script_launcher", "")
+            BufferedWriter(OutputStreamWriter(FileOutputStream(tmpFile))).use {
```

Comment:
Kotlin has its own versions of these functions:
```kotlin
val tmpFile = createTempFile("corda_node_script_launcher", "")
tmpFile.bufferedWriter().use {
   // etc
}
```
Don't forget to remove the unused import statements afterwards ... :wink: